### DISCUSSION TITLE
ASA Stats Engine v0.6.0 has been deployed!

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#371] [Announcements]
This update brings improvements in our caching mechanism that will allow bundles defined in our [Utilization document](https://github.com/asastats/channel/discussions/166) with up to 1000 Algorand addresses in them. Another server has been purchased for the purpose and now the primary/web server is used only for web page rendering, while most of the background tasks are moved to the second server.

This version brings an improved NFT engine too! :)  [ALGOxNFT](https://algoxnft.com/), [EXA Market](https://exa.market/), [Dartroom](https://dartroom.xyz/), and [ZestBloom](https://zestbloom.com/) are fully implemented. [FIFA+ Collect](https://collect.fifa.com/) purchases are added, while [Abris](https://abris.io/) purchases are improved. Non-ALGO listings and purchases are implemented too, for those NFT markets that support such functionality.

The next step is RandGallery and FIFA+ Collect listings. We have finally acquired Rand's API token, so its listings implementation is expected to be deployed this month.

Our current primary goal is to provide robust and trustworthy NFT API endpoints before 1/31. This version, which brings 95% of all ecosystem NFT purchases, is one of the final steps to achieve that goal. After we implement RandGallery and FIFA+ Collect listings we'll have around 95% of listings covered too, and the only significant task in front of us before releasing NFT beta (with prices evaluation) would be to implement various ecosystem swapping tools.

**Comment by Leonazirite** [comment]
Can we also get to see the current NFT market places where an NFT in an asset is trading?
It is pertinent to note that the last traded history of such NFT is already showing but the current trading place is not indicated. 
Thank you for the progress and development you bring to the Algorand world @ipaleka

**Reply by ipaleka** [reply]
We do track and link all the listed NFT of that particular address and also we list floor price together with a link to the NFT gallery.

RandGallery listings are going to be deployed this week, while FIFA+ Collect listings are planned in a week or so. If you are referring to some other NFT market/gallery then please create an issue here in this repo. ;)
